infrastructure/main.tf

```diff
@@ -96,6 +111,36 @@ resource "azurerm_key_vault_secret" "POSTGRES_DATABASE" {
   value        = "${module.db.postgresql_database}"
 }
 
+resource "azurerm_key_vault_secret" "postgres_user_v11" {
+  key_vault_id = "${module.send-letter-key-vault.key_vault_id}"
+  name         = "${var.component}-db-user-v11"
+  value        = "${module.db.user_name}"
```

Comment:
db-v11 should be used ?

Reply:
this pr is creating resources so the next pr can be using them via flux config ;)

Reply:
I mean should it be not `"${module.db-v11.user_name}` as you are populating new secrets ?

Reply:
oooh got you. yes yes yes! replacing names is not enough with this copy/paste. thanks